windows/installer.nsi

```diff
@@ -75,9 +75,9 @@ Page Custom ASIOCheckInstalled ExitASIOInstalled
 !insertmacro MUI_UNPAGE_FINISH
 
 ; Supported languages configuration
-; Additional languages can be added in the file installerlng.nsi in the translate folder, see https://nsis.sourceforge.io/Examples/Modern%20UI/MultiLanguage.nsi
+; Additional languages can be added in the file installerlng.nsi in the wininstaller folder, see https://nsis.sourceforge.io/Examples/Modern%20UI/MultiLanguage.nsi
 
-!include "translate\installerlng.nsi"
+!include "${ROOT_PATH}\src\res\translation\wininstaller\installerlng.nsi"
```

Comment:
Comparing with installerlng.nsi, that uses `..\src\res\...`, so should this be `..\src\...` instead of `${ROOT_PATH}\src\...`?

Reply:
I think ${ROOT_PATH} should also be used in the other file which would then make it an absolute path (as far as I remember these are the only relative paths in the installer).

Reply:
I wasn't sure, because all those paths were previously relative too (`translate\...`). I agree consistency would be good, one way or the other.

Reply:
Looking through the `.nsi` files, it appears the only other relative paths are referring to items not in the tree, such as `.nsh` files. So I would assume they are found via some include path. In view of that, I would agree that it would be safer for `installerlng.nsi` to refer to the individual language `.nsi` files using `${ROOT_PATH}\src\...` instead of `..\src\...`. It would avoid any ambiguity between "relative to the working directory" or "relative to the directory of the including file".